tests/scenario_tests/test_events.py

```diff
@@ -143,3 +143,21 @@ def handle_app_mention(body, say, payload, event):
         assert self.mock_received_requests["/auth.test"] == 1
         sleep(1)  # wait a bit after auto ack()
         assert self.mock_received_requests["/chat.postMessage"] == 1
+
+    def test_stable_auto_ack(self):
+        app = App(client=self.web_client, signing_secret=self.signing_secret)
+
+        @app.event("reaction_added")
+        def handle_app_mention():
+            raise Exception("something wrong!")
```

Comment:
Super innocuous, but maintaining the same capitalization as the other message introduced that has the same text? `"Something wrong!"` in `always_failing()` The other error messages I'm seeing at first glance also are sentence cased.

Reply:
We should do it. I will update the string value before merging this. Thanks!